hedera-node/hedera-app/src/main/java/com/hedera/node/app/state/merkle/disk/OnDiskValue.java

```diff
@@ -97,13 +96,16 @@ public VirtualValue asReadOnly() {
     /** {@inheritDoc} */
     @Override
     public void serialize(@NonNull final ByteBuffer byteBuffer) throws IOException {
+        throw new UnsupportedOperationException("Should never be called");
+        /*
```

Comment:
Probably should remove the commented-out code.

Reply:
Yes, I'll remove it